src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static animals(animal) {
+    Animal.alive.push(animal);
+  }
```

Comment:
I think that you don't need this

Reply:
That's better to work with static elements using static methods